### PR TITLE
[Feature] Fix README base's link

### PR DESCRIPTION
**Pull request by geoyee**
修改了案例的链接到md文件

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,9 +192,9 @@ PaddleRS是xxx、xxx、xxx等遥感科研院所共同基于飞桨开发的遥感
 * 模型训练
 * 推理部署
 * 应用案例
-  * [【官方】第十一届 “中国软件杯”大学生软件设计大赛—A组：变化检测](https://aistudio.baidu.com/aistudio/projectdetail/3684588)
-  * [使用超分模块提高真实的低分辨率无人机影像的分割精度](https://aistudio.baidu.com/aistudio/projectdetail/3696814)
-  * [无人机影像汽车识别](https://aistudio.baidu.com/aistudio/projectdetail/3713122/i)
+  * [【官方】第十一届 “中国软件杯”大学生软件设计大赛—A组：变化检测](./docs/cases/csc_cd_cn.md)
```
Comment (yzl19940819) [inline README.md]: 直接名字修改成《变化检测示例》或者xx区域xx-xx年的变化检测题目。

File: docs/cases/car_det_cn.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+基于0.5m的高分辨率无人机影像，我们希望能够使用目标检测的方法找到影像中的汽车。项目将基于PaddleRS完成该任务。
+
+## 1 数据准备
+
+数据来自于[DFC2018 Houston](https://hyperspectral.ee.uh.edu/?page_id=1075)，裁剪为1400张596x601大小的图块，由手工标注而成并按照9:1划分训练集和数据集。
+
+```python
+# 解压数据集
+! mkdir -p dataset
```
Comment (yzl19940819) [inline docs/cases/car_det_cn.md]: ！这个是aistudio的用户，需要去掉。

File: docs/cases/car_det_cn.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+! unzip -oq data/data56250/carDetection_RGB.zip -d dataset
```
Comment (yzl19940819) [inline docs/cases/car_det_cn.md]: 这里数据是aistudio的，只需要说明这个数据集解压就好

File: docs/cases/car_det_cn.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                    f_va.write(f"JPEGImages/{name}.jpg Annotations/{name}.xml\n")
+                else:
+                    f_tr.write(f"JPEGImages/{name}.jpg Annotations/{name}.xml\n")
+    labels = ["car"]
+    txt_str = "\n".join(labels)
+    with open((data_dir + "/" + f"label_list.txt"), "w") as f:
+        f.write(txt_str)
+    print("Finished!")
+
+get_data_list("dataset")
```
Comment (yzl19940819) [inline docs/cases/car_det_cn.md]: 这个示例可以写成一个py文件

File: docs/cases/car_det_cn.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+    pretrain_weights="COCO",
+    learning_rate=3e-5,
+    warmup_steps=10,
+    warmup_start_lr=0.0,
+    save_interval_epochs=5,
+    lr_decay_epochs=[10, 20],
+    save_dir="output",
+    use_vdl=True)
+```
+
```
Comment (yzl19940819) [inline docs/cases/car_det_cn.md]: 提供一个完整的py文件

File: docs/cases/car_det_cn.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+## 4 模型评估
+
+只需要调用evaluate即可完成预测。
+
```
Comment (yzl19940819) [inline docs/cases/car_det_cn.md]: eval的接口的参数在描述一下

File: docs/cases/car_det_cn.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+img_path = "dataset/JPEGImages/UH_NAD83_272056_3289689_58.jpg"
+pred = model.predict(img_path, eval_transforms)
+vis_img = visualize_detection(img_path, pred, save_dir=None)
+plt.figure(figsize=(10, 10))
+plt.imshow(vis_img)
+plt.show()
+```
+
+![output_13_0](https://user-images.githubusercontent.com/71769312/161358212-5f525ba3-059c-4c07-9d2e-ed4334069983.png)
+
```
Comment (yzl19940819) [inline docs/cases/car_det_cn.md]: 文档中同理，添加一个py文件，下同

**Review by yzl19940819 — COMMENTED** [review]
在改动一下

**Comment by geoyee** [comment]
了解，马上改下

File: docs/cases/car_det_cn.md (hunk 7 above)
Comment (geoyee) [reply docs/cases/car_det_cn.md]: 余哥我把这个删了吧，到时候直接更新`tutorials\train`的README吧，里面都是案例的py文件，晖哥已经提交过了看起来差不多，除了数据不太一样。以及没有eval之类的（`PaddleRS\tutorials\train\object_detection\ppyolov2.py`）
